scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Rather than writing "comparable" could you maybe say something like that the reported IDF1 is XX% and as one can see in the next section, our evaluation outputs an IDF1 which matches the reported accuracy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>